packages/server/src/nestjs/zenstack.module.ts

```diff
@@ -12,7 +12,7 @@ export interface ZenStackModuleOptions {
     /**
      * A callback for getting an enhanced `PrismaClient`.
      */
-    getEnhancedPrisma: () => unknown;
+    getEnhancedPrisma: (prop?: string | symbol) => unknown;
```

Comment:
Shall we rename `prop` to `model` instead so it's meaning is clearer? Also, I think in this case it's always typed as string. Maybe remove the `| symbol` part so the user can have a simpler callback signature?

Reply:
Sounds good, I'll update this in this tomorrow and push it.

Reply:
Thank you!

Reply:
So it looks like not including `symbol` throughs a type error. 

```
 Type 'symbol' is not assignable to type 'string'.
 ```
 https://github.com/zenstackhq/zenstack/actions/runs/9877836393/job/27280399900?pr=1538